streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -344,6 +342,30 @@ private void initTopology() {
         }
     }
 
+    private void closeTopology() {
```

Comment:
This is not intentional: I was re-arranging the public / protected / private functions together to analyze if any of them are not necessary; but this should not be part of a PR. Will revert.